src/main/java/seedu/address/logic/Messages.java

```diff
@@ -48,7 +48,7 @@ public class Messages {
     public static final String MESSAGE_EMPTY_DATE = "Date should not be blank!";
 
     public static final String MESSAGE_EMPTY_PERSON_LIST =
-            "Current person list is empty!\n" + "%1$s command must only be used on non-empty person list.";
+            "Current client list is empty!\n" + "%1$s command must only be used on non-empty client list.";
```

Comment:
Are we changing every "person" to "client"? That will require a lot of changes.

Reply:
I think that's what we agreed on, @beginner-web-developer updated the user guide to be client already.